modules/custom/az_dam/config/quickstart/node.type.ib_media.yml

```diff
@@ -0,0 +1,18 @@
+uuid: 54899adf-0d2e-4225-92e6-77e52aecec40
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.